app/queries/servers/post.ts

```diff
@@ -230,3 +232,56 @@ export const observeSavedPostsByIds = (database: Database, postIds: string[]) =>
         switchMap((prefs) => of$(new Set(prefs.map((p) => p.name)))),
     );
 };
+
+export const getIsPostPriorityEnabled = async (database: Database) => {
+    const featureFlag = await getConfigValue(database, 'FeatureFlagPostPriority');
+    const cfg = await getConfigValue(database, 'PostPriority');
+    return featureFlag === 'true' && cfg === 'true';
+};
+
+export const getIsPostAcknowledgementsEnabled = async (database: Database) => {
+    const cfg = await getConfigValue(database, 'PostAcknowledgements');
+    return cfg === 'true';
+};
+
+export const observeIsPostPriorityEnabled = (database: Database) => {
+    const featureFlag = observeConfigBooleanValue(database, 'FeatureFlagPostPriority');
+    const cfg = observeConfigBooleanValue(database, 'PostPriority');
+    return featureFlag.pipe(
+        combineLatestWith(cfg),
+        switchMap(([ff, c]) => of$(ff && c)),
+        distinctUntilChanged(),
+    );
+};
+
+export const observeIsPostAcknowledgementsEnabled = (database: Database) => {
+    return observeConfigBooleanValue(database, 'PostAcknowledgements');
+};
+
+export const observePersistentNotificationsEnabled = (database: Database) => {
+    const user = observeCurrentUser(database);
+    const enabledForAll = observeConfigBooleanValue(database, 'AllowPersistentNotifications');
+    const enabledForGuests = observeConfigBooleanValue(database, 'AllowPersistentNotificationsForGuests');
+    return user.pipe(
+        combineLatestWith(enabledForAll, enabledForGuests),
+        switchMap(([u, forAll, forGuests]) => {
+            if (u?.isGuest) {
+                return of$(forAll && forGuests);
+            }
+            return of$(forAll);
+        }),
+        distinctUntilChanged(),
+
+    );
+};
+
+export const getUsersCountFromMentions = async (database: Database, mentions: string[]) => {
+    const groupsQuery = queryGroupsByNames(database, mentions).fetch();
+    const usersQuery = queryUsersByUsername(database, mentions).fetchCount();
+    const [groups, usersCount] = await Promise.all([groupsQuery, usersQuery]);
+    let count = usersCount;
+    groups.forEach((group) => {
+        count += group.memberCount;
+    });
+    return count;
```

Comment:
```suggestion
    return groups.reduce((acc, v) => acc + v.memberCount, usersCount);
```
0/5 . Not sure which version has better legibility. Performance-wise probably are the same, so not really an issue there.

Reply:
I prefer the reduce :)